Packs/CiscoAMP/Integrations/CiscoAMP/CiscoAMP.py

```diff
@@ -0,0 +1,3528 @@
+            response=raw_response,
+            header_by_keys=VULNERABLE_COMPUTER_HEADERS_BY_KEYS,
+            keys_to_items_option_1=['data'],
+            title=VULNERABLE_COMPUTER_TITLE,
+        )
+
+    context_output = get_context_output(raw_response, ['links'])
+
+    return CommandResults(
+        outputs_prefix='CiscoAMP.Vulnerability',
+        outputs=context_output,
+        readable_output=readable_output,
+        raw_response=raw_response,
+    )
+
+
+def endpoint_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    """
+    Generic command that returns information about an endpoint.
+
+    Args:
+        client (Client): Cisco AMP client to run desired requests
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Raises:
+        DemistoException: Incase ID, IP or an hostname wasn't inserted
+
+    Returns:
+        List[CommandResults]: A list of endpoint indicators.
+    """
+    endpoint_ids = argToList(args.get('id'))
+    endpoint_ips = argToList(args.get('ip'))
+    endpoint_hostnames = argToList(args.get('hostname'))
+
+    if not endpoint_ids and not endpoint_ips and not endpoint_hostnames:
+        raise DemistoException('CiscoAMP - In order to run this command, please provide a valid id, ip or hostname')
+
+    responses = []
+
+    if endpoint_ids:
+        for endpoint_id in endpoint_ids:
+            response = client.computer_get_request(
+                connector_guid=endpoint_id
+            )
+
+            responses.append(response)
+
+    elif endpoint_ips:
+        for endpoint_ip in endpoint_ips:
+            response = client.computer_list_request(
+                internal_ip=endpoint_ip
+            )
+
+        responses.append(response)
+
+    else:
+        responses.append(client.computer_list_request(
+            hostnames=endpoint_hostnames
+        ))
+
+    endpoints: List = []
+
+    for response in responses:
+        data_list = response['data']
+
+        if endpoint_ids:
+            data_list = [data_list]
+
+        for data in data_list:
+            endpoint = Common.Endpoint(
+                id=data['connector_guid'],
+                ip_address=data['internal_ips'][0],
+                hostname=data['hostname'],
+                mac_address=data['network_addresses'][0]['mac'],
+                os=data['operating_system'],
+                os_version=data['os_version'],
+                status='Online' if data['active'] else 'Offline',
+                vendor='CiscoAMP Response'
+            )
+
+            endpoint_context = endpoint.to_context().get(Common.Endpoint.CONTEXT_PATH)
+            readable_output = tableToMarkdown(
+                f'CiscoAMP - Endpoint {data["hostname"]}',
+                endpoint_context
+            )
+
+            endpoints.append(CommandResults(
+                readable_output=readable_output,
+                outputs_prefix='',
```

Comment:
on purpose?

Reply:
Removed `outputs_prefix`, if that's what you mean.